crates/router/src/core/cache.rs

```diff
@@ -10,19 +10,12 @@ pub async fn invalidate(
     key: &str,
 ) -> CustomResult<services::api::ApplicationResponse<serde_json::Value>, errors::ApiErrorResponse> {
     let store = state.store.as_ref();
-    let result = publish_into_redact_channel(
+    publish_into_redact_channel(
```

Comment:
We'd end up leaving the Redis value behind. Should we consider invalidating that as well?